### PR TITLE
CAD-763 adapt configuration for LiveView

### PR DESCRIPTION
**Pull request by CodiePP**
Issue
-----------

* configuration adaptations:
  - comment out the default log output to 'stdout'
  - set output mode to LiveView

* launch the TUI with script: `./scripts/mainnet.sh`
  (add `--stack-nix` or other options)

- This PR **does not result** in breaking changes to upstream dependencies.

Checklist
---------
- [x] This PR contains all the work required to resolve the linked issue.

- [x] The work contained has sufficient documentation to describe what it does and how to do it.

- [ ] The work has sufficient tests and/or testing.

- [ ] I have committed clear and descriptive commits. Be considerate as somebody else will have to read these.

- [x] I have added the appropriate labels to this PR.

### CODE REVIEW

File: configuration/defaults/mainnet/configuration.yaml [hunk 1]
```diff
@@ -220,21 +222,27 @@ options:
   # and override and not an extension so anything matched here will not
   # go to the default backend, only to the explicitly listed backends.
   mapBackends:
+    cardano.node.BlockFetchDecision.peers:
+      - TraceForwarderBK
+      - EKGViewBK
+      - kind: UserDefinedBK
+        name: LiveViewBackend
```
Comment (dcoutts) [inline configuration/defaults/mainnet/configuration.yaml:229]: We have no setup above for a `UserDefinedBK`. How does this interact with the `setupBackends` directive?

File: configuration/defaults/mainnet/configuration.yaml [hunk 2]
```diff
@@ -220,21 +222,27 @@ options:
   # and override and not an extension so anything matched here will not
   # go to the default backend, only to the explicitly listed backends.
   mapBackends:
+    cardano.node.BlockFetchDecision.peers:
+      - TraceForwarderBK
+      - EKGViewBK
```
[the remark this answers is not in the document]
Comment (CodiePP) [reply configuration/defaults/mainnet/configuration.yaml:227]: yes, if the backend is not setup then it won't be used by the switchboard.

File: configuration/defaults/mainnet/configuration.yaml (hunk 1 above)
Comment (CodiePP) [reply configuration/defaults/mainnet/configuration.yaml:229]: this is the LiveView backend that will be setup in the code depending on the switch in the configuration.

File: configuration/defaults/mainnet/configuration.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dcoutts) [reply configuration/defaults/mainnet/configuration.yaml:227]: Ok, then we should have a commented out entry for `TraceForwarderBK` so people know how to enable it.

[the remark this answers is not in the document]
Comment (CodiePP) [reply configuration/defaults/mainnet/configuration.yaml:243]: the `TraceForwarderBK` is setup when the configuration contains a definition of `forwardTo`. I hope the description is now much clearer on this.

**Pull request merged**